processors/filterprocessor/gen/generate.go

```diff
@@ -0,0 +1,145 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"go/format"
+	"os"
+	"reflect"
+	"sort"
+	"strings"
+
+	"github.com/algorand/go-algorand/data/transactions"
+)
+
+// recursiveTagFields recursively gets all field names in a struct
+// Output will contain a key of the full tag along with the fully qualified struct
+func recursiveTagFields(theStruct interface{}, Output *map[string]string, tagLevel []string, fieldLevel []string) {
```

Comment:
Does capitalizing `Output` provide any functional difference or follow some golang convention? A minute of googling didn't reveal anything to me.

Reply:
I believe it is more idiomatic to pass maps directly as they are reference types, so no need to pass a pointer to a `map[string]string`.

Reply:
Done